package.json

```diff
@@ -37,7 +37,8 @@
   "devDependencies": {
     "@alephium/sdk": "0.5.0",
     "@alephium/walletconnect-provider": "0.2.1",
-    "@alephium/web3": "0.2.0",
+    "@alephium/web3": "0.5.2",
```

Comment:
This is a simple upgrade of the library, the rest of the diff is just automatic reordering after running `npm i`.